backend/src/ee/services/secret-approval-policy/secret-approval-policy-service.ts

```diff
@@ -353,7 +353,7 @@ export const secretApprovalPolicyServiceFactory = ({
     );
     ForbiddenError.from(permission).throwUnlessCan(
       ProjectPermissionActions.Read,
-      subject(ProjectPermissionSub.Secrets, { secretPath, environment })
+      subject(ProjectPermissionSub.Secrets, { secretPath, environment, secretName: "", secretTags: [] })
```

Comment:
This feels like it should stem from a different permission subject because secret name and tags don't really apply here?

Reply:
To an extend I don't even know why we have a rule there. `GetApprovalPolicy` is not even a sensitive thing